Include/internal/pycore_bitutils.h

```diff
@@ -131,8 +131,40 @@ _Py_popcount32(uint32_t x)
 }
 
 
+// Return the index of the most significant 1 bit in 'x'. This is the smallest
+// integer k such that x < 2**k. Equivalent to floor(log2(x)) + 1 for x != 0.
+static inline int
+_Py_bit_length(unsigned long x) {
```

Comment:
nitpick: PEP 7 coding style:

```suggestion
_Py_bit_length(unsigned long x)
{
```